styles/prosilver/template/contributions/contribution_header.html

```diff
@@ -1,16 +1,5 @@
 <!-- INCLUDE overall_header.html -->
 
-<!-- IF U_DOWNLOAD and U_SUBSCRIBE and not IS_SUBSCRIBED and not QUEUE_STATUS -->
```

Comment:
Because this is essentially dead code. From what I have tried to infer, this appeared to be part of an option to Subscribe to a contribution when you have chosen to download one. There is no more code in Titania that unhides this...nor I have I ever remembered experiencing such a thing.